### ISSUE TITLE
Evaluate Apple Silicon package

### ISSUE THREAD

**Issue by dvselas** [brutella/hkknx-public#83]
It would be great to run hkknx natively on Apple M1 chips as soon as go is available on Apple silicon.

**Comment by brutella** [comment]
There is now a pre-release build for version 1.3.0 available for M1 Macs (`..._darwin_arm64.tar.gz`).

**Comment by dvselas** [comment]
great news. thank you.

**Comment by brutella** [comment]
Have you tried the provided Apple silicon package? Is this issue now resolved?

**Comment by dvselas** [comment]
Yes I tried it, but i had some stability issues with 1.3.0 in general. Overall the build was running. 
The issue was that HomeKit/hkknx accepted the command but the devices did not change the state. If i remember correctly is saw write timeouts in the logs. I saw the same behaviour with another build on a RPi 4. That is why i moved back to 1.2.0. Unfortunately I had not more time to dig deeper.

Nevertheless we can close this issue as the build was ok. If I find some time I will try 1.3 again and report any issues i might find. At the moment I cannot finally tell if it was a bug or if it was because of my setup.

thank you for this great piece of software.
